main.css

```diff
@@ -0,0 +1,1029 @@
+    select.has-error:hover,
+    select.has-error:focus,
+    select.has-error:active,
+    textarea.has-error,
+    textarea.has-error:hover,
+    textarea.has-error:focus,
+    textarea.has-error:active {
+    border: 1px solid var(--primary-red);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.1), 0 0 6px #f4cecf; }
+    
+    input.is-success, input.is-success:hover, input.is-success:focus, input.is-success:active,
+    select.is-success,
+    select.is-success:hover,
+    select.is-success:focus,
+    select.is-success:active,
+    textarea.is-success,
+    textarea.is-success:hover,
+    textarea.is-success:focus,
+    textarea.is-success:active {
+    border: 1px solid var(--primary-green);
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.1), 0 0 6px #97efc0; }
+    
+    
+    /* forms **********************************************/
+    
+    
+    /* typography *********************************************/
+    h1,
+    h2,
+    h3,
+    h4,
+    h5 {
+    margin: 1.5rem 0;
+    font-weight: 600;
+    font-family: var(--font-family-sans-serif);
+    line-height: 1.2;
+    color: #404040; 
+    }
+    
+    h1:not(:first-child),
+    h2:not(:first-child),
+    h3:not(:first-child),
+    h4:not(:first-child),
+    h5:not(:first-child) {
+    margin: 1.5rem 0; }
+    
+    h1:not(:first-child),
+    h2:not(:first-child),
+    h3:not(:first-child) {
+    margin-top: 2rem; }
+    
+    h1 {
+    font-size: 3rem; }
+    
+    h2 {
+    font-size: 2.5rem; }
+    
+    h3 {
+    font-size: 2rem; }
+    
+    h4 {
+    font-size: 1.5rem; }
+    
+    h5 {
+    font-size: 1rem; }
+    
+    
+    /* text alignment */
+    
+    .text-left {
+    text-align: left; }
+    
+    .text-right {
+    text-align: right; }
+    
+    .text-center {
+    text-align: center; }
+    
+    .text-justify {
+    text-align: justify; }
+    
+    
+    /* mute text */
+    
+    .text-mute{
+      color: var(--gray);
+    }
+    
+    .text-sm{
+      font-size: small;
+    }
+    
+    /* typography *********************************************/
+    
+    
+    /* list **************************************************/
+    
+    dt {
+      font-weight: 600; }
+    
+    dd {
+      margin-bottom: 0.5rem; }
+    
+    /* list **************************************************/
+    
+    
+    /* flex grid */
+    
+    
+    .flex-small,.flex-big,.flex-medium{
+      flex-basis: 100%;
+      padding: 0 1rem 0;
+      margin-bottom: 1rem;
+    }
+      
+      /* Small screen breakpoint */
+      @media (min-width: 600px) {
+      .flex-small {
+      flex: 1;
+      margin-bottom: 0; }
+      .flex-small.half {
+      flex: 0 0 calc(1 / 2 * 100%); }
+      .flex-small.one-fourth {
+      flex: 0 0 calc(1 / 4 * 100%); }
+      .flex-small.three-fourths {
+      flex: 0 0 calc(3 / 4 * 100%); }
+      .flex-small.one-third {
+      flex: 0 0 calc(1 / 3 * 100%); }
+      .flex-small.two-thirds {
+      flex: 0 0 calc(2 / 3 * 100%); } }
```

Comment:
code formatting can be done here

Reply:
grid is yet to done completely ..will do it